packages/cli/src/commands/addons/create.ts

```diff
@@ -68,13 +68,19 @@ export default class Create extends Command {
   }
 
   public async run(): Promise<void> {
+    this.allowArbitraryFlags = true
     const {flags, args, ...restParse} = await this.parse(Create)
     const {app, name, as, wait, confirm} = flags
     const servicePlan = args['service:plan']
     const argv = (restParse.argv as string[])
     // oclif duplicates specified args in argv
       .filter(arg => arg !== servicePlan)
 
+    if (restParse.nonExistentFlags && restParse.nonExistentFlags.length > 0) {
+      ux.log(` ${color.yellow('›')}   For example: ${color.cyan(`heroku addons:create -a ${app} ${restParse.raw[0].input} -- ${restParse.nonExistentFlags.join(' ')}`)}`)
```

Comment:
The reason this is `ux.log()` and not `ux.warn()` is because when the full warning displays when the deprecated syntax is used, it will show a duplicate of the ` >   Warning:` styling. This is because part of the full deprecation warning is located in the command specifically, not in the `heroku-cli-command` [deprecation warning](https://github.com/heroku/heroku-cli-command/pull/139). This is also reflected in the added tests since the outputs are from 2 different streams (`stdout` and `stderr`). There is no other way to get the same warning UI without repeating the ` >   Warning:` styling that I'm aware of. I'm open to other ways of implementing if possible!

Reply:
Fixed this issue with `process.stderr.write()`.